### ISSUE TITLE
False positive value-keyword-case with native value

### ISSUE THREAD

**Issue by rohm1** [stylelint/stylelint#6883]
### What steps are needed to reproduce the bug?

```css
* {
	text-rendering: optimizeLegibility;
}
```

### What Stylelint configuration is needed to reproduce the bug?

```json
{
  "extends": [
    "stylelint-config-standard"
  ]
}
```

### How did you run Stylelint?

Demo website: https://stylelint.io/demo/#N4Igxg9gJgpiBcIBUACYAdAdugLjmAHjgLQBOMmspAlpgObwoQAOO1AttQF4wAyMdagCNqAG2o4AngG4sAXxAAaEADMxMAHIBDdnESEdzUTAB0YAM7ml4CJjV0EIDJhQp0IQvkrn3jANpYrq7u5lLG4pgkkHbUdMShWpRapFDugSgAuvLW0fYAYhCk7Fo4jgBW5rbWsMxWiM7BIKGS4bQ4vm4goiUwoe6K6SFhMBFRtvbxOIlQyakInd34fSDZckA

### Which version of Stylelint or dependencies are you using?

latest

### What did you expect to happen?

No problems to be reported

### What actually happened?

Error: Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

### Does the bug relate to non-standard syntax?

no

### Proposal to fix the bug

Possibly related to #6517

**Comment by ybiquitous** [comment]
@rohm1 Thanks for the report. But I believe this is not a bug. You can avoid the error by the [`camelCaseSvgKeywords`](https://stylelint.io/user-guide/rules/value-keyword-case/#camelcasesvgkeywords-true--false-default-false) option:

```json
{
  "rules": {
    "value-keyword-case": ["lower", {"camelCaseSvgKeywords": true}]
  }
}
```

See also [this demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIBUACYAdAdugLjmAHjgLQBOMmspAlpgObwoQAOO1AttQF4wAyMdagCNqAG2o4AngG4sAXxAAaEADMxMAHIBDdnESEdzUTAB0YAM7ml4CJjV0EIDJhQp0IQvkrn3jANpYrq7u5lLG4pgkkHbUdMShWpRapFDugSgAuorp7qQArsY+CGjpwSAAblqieTDEANYwkgDuECnEYFrmML4oASCiEE0wpO6KJeA6MKIAwp0wAMrldADSjS0pRYw4+TByGely8tbR9gBirexaOI4AVua21rDMVojOZaGS4bQ4Pe6iVzBQqMciAPl9Iu1bPZ4jhElBkqlin8AUCQEc5EA).

**Comment by rohm1** [comment]
@ybiquitous thanks, but this is at least a default config bug. I should not have to configure anything for standard CSS properties and values.

**Comment by ybiquitous** [comment]
Make sense. Let's move this into the standard config repo and continue discussing.

Ref: https://github.com/stylelint/stylelint-config-standard/blob/5aa5dd57ac814edd98c656da35f2ccc99a64d433/index.js#L111
